docs/ONNX_Runtime_Perf_Tuning.md

```diff
@@ -70,30 +70,32 @@ sess_options.intra_op_num_threads=2
 sess_options.enable_sequential_execution=True
```

Comment:
```
sess_options.enable_sequential_execution=True
```
Pranav is changing the API. When his has done the work, we should review this part and refresh it again.

Reply:
CC @pranavsharma please note and include this update as part of your PR